js/models/entity.js

```diff
@@ -0,0 +1,10 @@
+class Entity{
```

Comment:
we should probably namespace our stuff at some point instead of overloading the window space. For now this is ok.